src/elements/_has-arrow.scss

```diff
@@ -15,7 +15,7 @@ $arrow-box-side: 13px;
 // Calculate long side (i.e., the hypotenuse of the arrow container)
 //$arrow-hypotenuse: sqrt(2 * $arrow-box-side * $arrow-box-side); // 18.3848px
 // sqrt(2 * boxSide * boxSide) = sqrt(2 * 13 * 13) = sqrt(338) = 18.3848px
-$arrow-hypotenuse: 18.3848px;
+$arrow-hypotenuse: 18.385px;
```

Comment:
was this due to linting? should you update the comments above to reflect the rounding? not everyone rounds automatically in their head

Reply:
yeah, this is due to the linting rule `"number-max-precision": 3`

Reply:
yeah, this is due to the linting rule `"number-max-precision": 3`

Reply:
If we had SASS doing the calculations, this wouldn't be an issue, but we have to hard code it for now, because SASS doesn't have built-in functions for `pow()` or `sqrt()`